flexgen/apps/task.yaml

```diff
@@ -0,0 +1,35 @@
+# benchmark.yaml
```

Comment:
Consider renaming this file to `skypilot.yaml`? To avoid task vs. job ambiguity.